tests/cypress/tests-e2e/timeSlider.cy.js

```diff
@@ -30,16 +30,18 @@ describe('Testing the time slider', () => {
             cy.get('[data-cy="times-slider-cursor"]').trigger('mouseup', { force: true })
         }
         const preSelectedYear = 2019
-        beforeEach(() => {
+        it('should have the preselected year correctly set', () => {
             cy.goToMapView({
                 layers: `test.timeenabled.wmts.layer@year=${preSelectedYear}`,
             })
             cy.get('[data-cy="time-slider-button"]').click()
-        })
-        it('should have the preselected year correctly set', () => {
             cy.get('[data-cy="time-slider-current-year"]').should('contain', preSelectedYear)
         })
-        it('should change the year if the user drags the tooltip on the right with the mouse', () => {
+        it('should show the slider on startup when setting it in the URL, and change the year if the user drags the tooltip on the right with the mouse', () => {
+            cy.goToMapView({
+                layers: `test.timeenabled.wmts.layer`,
+                timeSlider: preSelectedYear,
+            })
             moveSlider(200)
             cy.get('[data-cy="time-slider-current-year"]').should('not.contain', preSelectedYear)
         })
```

Comment:
This should be/could be merged together. Also you should add more test case in e2e tests for the timeslider. Same when moving the slider we should make sure that the time is in the active layer list selected, and that non visible layer are not changed. Also make sure that the timeSlider url param has been changed, etc..